src/main/java/uk/gov/digital/ho/hocs/info/domain/repository/CaseTypeRepository.java

```diff
@@ -10,9 +10,8 @@
 @Repository
 public interface CaseTypeRepository extends CrudRepository<CaseType, String> {
 
-    // Behaviour is to list both cases with NULL previousCaseType + any NOT NULL when asked for
-    @Query(value = "SELECT ct.* FROM case_type ct JOIN permission ON ct.type = permission.case_type WHERE permission.team_uuid IN ?1 AND ct.active AND (ct.previous_case_type IS NULL OR (ct.previous_case_type IS NOT NULL) = ?2)", nativeQuery = true)
-    Set<CaseType> findAllCaseTypesByTeam(Set<String> team, boolean includePreviousCaseType);
+    @Query(value = "SELECT ct.* FROM case_type ct JOIN permission ON ct.type = permission.case_type WHERE permission.team_uuid IN ?1 AND ct.active = TRUE", nativeQuery = true)
+    Set<CaseType> findAllCaseTypesByTeam(Set<String> team);
```

Comment:
This change presents the correct behaviour for my scenario but can who PR's this take another critical look at it because I wasn't 100% convinced by it but couldn't find an immediate detrimental consequence.

Reason for the change was without it the case type wasn't presented in the Create Single Case list of available case types.